sklearn/inspection/_permutation_importance.py

```diff
@@ -143,14 +175,26 @@ def permutation_importance(estimator, X, y, *, scoring=None, n_repeats=5,
     random_state = check_random_state(random_state)
     random_seed = random_state.randint(np.iinfo(np.int32).max + 1)
 
-    scorer = check_scoring(estimator, scoring=scoring)
-    baseline_score = _weights_scorer(scorer, estimator, X, y, sample_weight)
-
-    scores = Parallel(n_jobs=n_jobs)(delayed(_calculate_permutation_scores)(
-        estimator, X, y, sample_weight, col_idx, random_seed, n_repeats, scorer
-    ) for col_idx in range(X.shape[1]))
-
-    importances = baseline_score - np.array(scores)
-    return Bunch(importances_mean=np.mean(importances, axis=1),
-                 importances_std=np.std(importances, axis=1),
-                 importances=importances)
+    if scoring is None or isinstance(scoring, str) or callable(scoring):
+        scorer = check_scoring(estimator, scoring=scoring)
+    else:
+        scorers_dict = _check_multimetric_scoring(estimator, scoring)
+        scorer = _MultimetricScorer(**scorers_dict)
+
+    baseline_score = _weights_scorer(scorer, estimator, X, y,
+                                     sample_weight)
+
+    scores = Parallel(n_jobs=n_jobs)(
+        delayed(_calculate_permutation_scores)(
+            estimator, X, y, sample_weight, col_idx, random_seed,
+            n_repeats, scorer
+        ) for col_idx in range(X.shape[1]))
+
+    if isinstance(baseline_score, dict):
+        ret = dict()
+        for name in baseline_score:
+            score = [scores[col_idx][name] for col_idx in range(X.shape[1])]
+            ret[name] = _create_importances_bunch(baseline_score[name], score)
+        return ret
+    else:
+        return _create_importances_bunch(baseline_score, score)
```

Comment:
```suggestion
        return _create_importances_bunch(baseline_score, np.array(score))
```

Reply:
I put the `np.array(` inside  `_create_importances_bunch`. Should do otherwise?

Reply:
got it in the next comment